docs/lib/examples/NavbarToggler.js

```diff
@@ -21,18 +20,18 @@ export default class Example extends React.Component {
     return (
       <div>
         <Navbar color="faded" light>
-          <NavbarBrand href="/">reactstrap</NavbarBrand>
           <NavbarToggler className="float-sm-right hidden-lg-up collapsed" onClick={this.toggleNavbar} />
-          <div className={classnames('collapse', 'navbar-toggleable-md', {in: !this.state.collapsed})}>
-            <Nav className="float-sm-left float-md-right" navbar>
+          <Collapse className="navbar-toggleable-md" isOpen={!this.state.collapsed}>
+            <NavbarBrand href="/">reactstrap</NavbarBrand>
```

Comment:
For sure. I'm holding off on documenting custom styles but I might do something like [this](http://codepen.io/eddywashere/pen/wGVgoN) for the docs site navbar. Thanks again for documenting this!